tsconfig.json

```diff
@@ -21,5 +21,8 @@
   },
   "include": [
     "src"
+  ],
+  "typeRoots": [
+    "./node_modules/@types", "./node_modules/jest-enzyme"
```

Comment:
Do we know what this line does exactly? Just want to make sure we haven't added anything we don't need to 😄

Reply:
Isn't that the typescript types for the jest module?

Reply:
@danielwilliams94 well spotted actually, [type root](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html) specifies that only those types are included and not anything else.